src/assets/scss/components/button.scss

```diff
@@ -1,11 +1,11 @@
 @import 'src/colors.scss';
 @import 'src/geometry.scss';
 
-$secondary-color: #4a4a4a;
-$warning-color: #f5a623;
-$danger-color: #ff001f;
-
 .alg-button {
+  --secondary-color: #4a4a4a;
+  --warning-color: #f5a623;
+  --danger-color: #ff001f;
```

Comment:
Is it normal to have such variables declared at a button-level ?
I would expect colors to be declared at root or theme level :thinking:

Reply:
It's not normal, but it was done before long time ago. I guess refactoring scss approach is out of task scope, otherwise need additional time for investigate why it was done like this, then fix, then tested.

Reply:
Ok, can you check if there's a refactoring task for that and if not create it ?

Reply:
Fixed. Issue #930